docs/best_practices/time_series.rst

```diff
@@ -107,6 +107,23 @@ Check function: :py:meth:`~nwbinspector.checks.time_series.check_regular_timesta
 
 
 
+
+.. _best_practice_timestamp_of_the_first_sample:
+
+Timestamp of the first sample
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The timestamp of the first sample of a :ref:`nwb-schema:sec-TimeSeries` defined by
+:ref:`nwb-schema:TimeSeries.starting_time <sec-TimeSeries>` or the first value in
+:ref:`nwb-schema:TimeSeries.timestamps <sec-TimeSeries>` should be `0.0` or any positive value.
+If you have multiple modalities with different timestamps, the earliest timestamp should
```

Comment:
I would say that this section can be condensed to a single line addition to https://github.com/catalystneuro/nwbinspector/blob/add_check_for_timestamps_are_not_negative/docs/best_practices/nwbfile_metadata.rst#global-date-and-time-reference

along the lines of "The `timestamps_reference_time` should also be the earliest timestamp in the file, giving all other time references a positive value relative to that. There should be no time references which are negative."